lesson4/public/index28.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title></title>
+</head>
+<body>
+  <form action= '/send'>
+  Name:<input type='text' name="a1"/>
+  Comment:<input type='text' name="a2"/>
```

Comment:
change the id of name and comment. make them a bit realistic instead of a1 and a2.

Reply:
done!